kiva/fonttools/_util.py

```diff
@@ -44,10 +44,80 @@
     5: "version",
     6: "postscript_name",
 }
+# OpenType 'OS/2' `ulCodePageRange` bit meanings
```

Comment:
can you add a reference here - something like this https://docs.microsoft.com/en-us/typography/opentype/spec/os2#cpr

Reply:
Yeah, I was thinking the same